quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImpl.java

```diff
@@ -67,5 +67,21 @@ public Collection editCollection(String id, String name, String description) thr
             throw new CollectionNotFoundException();
         }
     }
+    
+    /**
+     * Deletes a collection from mongodb collection table by _id
+     * 
+     * @param id used to look up the collection in mongodb
+     * 
+     */
+    public void deleteCollection(String id) throws CollectionNotFoundException {
+        Collection collection = findCollection(id);
+        
+        if(Objects.nonNull(collection)) {
+            collectionRepo.delete(collection);
+        } else {
+            throw new CollectionNotFoundException();
```

Comment:
In case if the collection does not exist, we would just return instead of throwing an exception. Could you verify @jdamerow .

Reply:
either approach is fine with me; you can either argue that deleting something that doesn't exist (aka is already deleted) should just behave like it's successfully deleted as the outcome is the same. Or you can say trying to delete something that doesn't exist should be made known. Either is fine with me as long as it's consistent across the application (meaning other types of objects should behave the same way).

Reply:
In that case, current version should be fine